build/const.js

```diff
@@ -0,0 +1,4 @@
+import path from 'path';
```

Comment:
s/const/constants to be consistent with other similar files, and remove ambiguity of JS const